deepsearch/documents/core/main.py

```diff
@@ -25,32 +30,50 @@ def convert_documents(
     url : string [OPTIONAL]
     For converting a document from the web, please provide its url.
 
-    local_file : path [OPTIONAL]
+    source_file : path [OPTIONAL]
     For converting local files, please provide absolute path to file or to directory
     containing multiple files.
 
-    NOTE: Either url or local_file should be supplied.
+    progress_bar : Boolean (default is False in code, True in CLI)
+    Show progress bar for processing, submitting, converting input and
+    downloading converted document.
+
+    cli_use : Boolean (default is False in code, True in CLI)
+    A flag that allows automatic download of converted document and
+    generation of conversion report when used via CLI.
+
+    NOTE: Either url or source_path should be supplied.
     """
 
     # initialize default Api if not specified
     if api is None:
         api = CpsApi.default_from_env()
 
     # check required inputs are present
-    if url is None and local_file is None:
+    if url is None and source_path is None:
         raise ValueError(
             "No input provided. Please provide either a url or a local file for conversion."
         )
-    elif url is not None and local_file is None:
-        if isinstance(url, str):
+    elif url is not None and source_path is None:
+        if urllib.parse.urlparse(url).scheme in ("http", "https"):
```

Comment:
What is this supposed to do? Accept URLs with `file://` or such?
Any valid URL must be using `http` or `https` protocol handler, so please remove the extra if/else.